kythe/java/com/google/devtools/kythe/platform/java/filemanager/CompilationUnitPathFileManager.java

```diff
@@ -58,10 +58,11 @@
 
   private final CompilationUnitFileSystem fileSystem;
   private final ImmutableSet<String> defaultPlatformClassPath;
-  // The path given to us that we are allowed to write in.
+  // The path given to us that we are allowed to write in. This will be stored as an absolute
+  // path.
   private final @Nullable Path temporaryDirectoryPrefix;
   // A temporary directory inside of temporaryDirectoryPrefix that we will use and delete when the
-  // close method is called.
+  // close method is called. This should be will as an absolute path.
```

Comment:
will be stored

Reply:
https://github.com/kythe/kythe/pull/4224